pkg/fd/fd.go

```diff
@@ -0,0 +1,109 @@
+//go:build !windows
+// +build !windows
```

Comment:
You can drop these lines if you rename the files to `*_unix.go`

Reply:
Moved to fork repo